src/components/App.css

```diff
@@ -1,3 +1,201 @@
-.App {
-  text-align: left;
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&family=Roboto+Slab:wght@100;200;300&display=swap');
+
```

Comment:
- [ ] Kindly use the same background-color for the body of your app. 

![Capture](https://user-images.githubusercontent.com/64173189/161357119-ed4e0113-1925-4f7d-825e-3a46db6ab3cb.JPG)

As you can see from the original design the background is not white. Please fix this minor issue.